feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -143,6 +144,11 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
                 binding.isCollapsed = isCollapsed
             }
         }
+        sessionPagesStore.reselectedTab.observe(viewLifecycleOwner) {
```

Comment:
Probably you can use `sessionPagesStore.reselectedTab.nonNull().`.

Reply:
`if statement` is  added in 124a068, so if `it` is null that will be no operation.